Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -191,8 +191,6 @@ void WebFoundTextRangeController::didBeginTextSearchOperation()
 
 void WebFoundTextRangeController::didEndTextSearchOperation()
```

Comment:
Nit - can we delete this method altogether?

Reply:
Yeah, removed this, as well as the plumbing around it. Thanks for the review!